### PR TITLE
chore: add Elena to Rust libp2p maintainers

### PR DESCRIPTION
**Pull request by jxs**
As she's back contributing with us.
cc @guillaumemichel @dariusc93

### CODE REVIEW

**Comment by galargh** [comment]
A sync failed yesterday, I'm rerunning it now. After that, the plan will rerun here.

File: github/libp2p.yml [hunk 1]
```diff
@@ -5472,6 +5472,7 @@ teams:
     members:
       maintainer:
         - AgeManning
+        - elenaf9
```
Comment (p-shahi) [inline github/libp2p.yml:5475]: you have to remove Elena's username from the `member` list @jxs

Comment (galargh) [reply github/libp2p.yml:5475]: This is correct. The other issue that I mentioned - the one with sync - is already resolved.

Comment (jxs) [reply github/libp2p.yml:5474]: thanks! Updated

**Review by guillaumemichel — APPROVED** [review]

**Review by dariusc93 — APPROVED** [review]

**Review by p-shahi — APPROVED** [review]

**Pull request merged**